main.go

```diff
@@ -109,12 +113,29 @@ func main() {
 			metric.WithRequestDurationBucket(prometheus.ExponentialBuckets(0.05, 2, 8))))
 	service.WithLogger(goalogrus.New(log.Logger()))
 
-	// service.Use(metric.Recorder())
+	appDB := application.NewGormApplication(db)
+	tokenManager, err := token.NewManager(config)
```

Comment:
Still would use `token.DefaultManager(config)` instead :) https://github.com/fabric8-services/fabric8-common/blob/master/token/token.go#L43
Both options will work but it's a good practice to initiate the default manager in main().
In this case you will have the same token manager instance used in all request contexts and get it available even if you don't have any context at all (for example if you need to start some goroutine to do some backround work and you need a token manager for something).

See example here:
https://github.com/fabric8-services/fabric8-cluster/blob/master/main.go#L152

Reply:
ok, let me apply that change